deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -1435,7 +1435,7 @@ metadata:
       \                   \"resources\": {}\n                }\n            ]\n        }\n
       \   }\n\t"
     operatorframework.io/suggested-namespace: openshift-storage
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
```

Comment:
Are we going backword in operator-sdk versions?

Reply:
No this is latest one

Reply:
We can only use sdk v1.13.1 or older due to K8s deps.